lib/team_factory.rb

```diff
@@ -16,4 +16,9 @@ def create_teams
     end
     @teams
   end
+  
+  def count_of_teams
```

Comment:
Yes we can talk about abstracting methods into their own respective classes! But this works in the TeamFactory class for now! Very nice